src/Goerli-DssSpell.sol

```diff
@@ -37,15 +37,98 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
     //
+    uint256 internal constant ONE_FIVE_PCT_RATE = 1000000000472114805215157978; 
 
     function officeHours() public override returns (bool) {
         return false;
     }
 
     function actions() public override {
+
+        // ---------------------------------------------------------------------
+        // Collateral Auction Parameter Changes
+        // https://vote.makerdao.com/polling/QmREbu1j#vote-breakdown
+        // https://forum.makerdao.com/t/collateral-auctions-analysis-parameter-updates-september-2022/18063
+        
+        // buf changes (Starting auction price multiplier)
+        DssExecLib.setStartingPriceMultiplicativeFactor("ETH-A"          , 110_00);
+        DssExecLib.setStartingPriceMultiplicativeFactor("ETH-B"          , 110_00);
+        DssExecLib.setStartingPriceMultiplicativeFactor("ETH-C"          , 110_00);
+        DssExecLib.setStartingPriceMultiplicativeFactor("WBTC-A"         , 110_00);
+        DssExecLib.setStartingPriceMultiplicativeFactor("WBTC-B"         , 110_00);
+        DssExecLib.setStartingPriceMultiplicativeFactor("WBTC-C"         , 110_00);
+        DssExecLib.setStartingPriceMultiplicativeFactor("WSTETH-A"       , 110_00);
+        DssExecLib.setStartingPriceMultiplicativeFactor("WSTETH-B"       , 110_00);
+        //DssExecLib.setStartingPriceMultiplicativeFactor("CRVV1ETHSTETH-A", 120_00); // Not on Goerli
+        DssExecLib.setStartingPriceMultiplicativeFactor("LINK-A"         , 120_00);
+        DssExecLib.setStartingPriceMultiplicativeFactor("MANA-A"         , 120_00);
+        DssExecLib.setStartingPriceMultiplicativeFactor("MATIC-A"        , 120_00);
+        DssExecLib.setStartingPriceMultiplicativeFactor("RENBTC-A"       , 120_00);
+        
+        // cusp changes (Max percentage drop permitted before auction reset)
+        DssExecLib.setAuctionPermittedDrop("ETH-A"    , 45_00);
+        DssExecLib.setAuctionPermittedDrop("ETH-B"    , 45_00);
+        DssExecLib.setAuctionPermittedDrop("ETH-C"    , 45_00);
+        DssExecLib.setAuctionPermittedDrop("WBTC-A"   , 45_00);
+        DssExecLib.setAuctionPermittedDrop("WBTC-B"   , 45_00);
+        DssExecLib.setAuctionPermittedDrop("WBTC-C"   , 45_00);
+        DssExecLib.setAuctionPermittedDrop("WSTETH-A" , 45_00);
+        DssExecLib.setAuctionPermittedDrop("WSTETH-B" , 45_00);
+        
+        // tail changes (Max auction duration)
+        DssExecLib.setAuctionTimeBeforeReset("ETH-A",    7200 seconds);
+        DssExecLib.setAuctionTimeBeforeReset("ETH-C",    7200 seconds);
+        DssExecLib.setAuctionTimeBeforeReset("WBTC-A",   7200 seconds);
+        DssExecLib.setAuctionTimeBeforeReset("WBTC-B",   7200 seconds);
+        DssExecLib.setAuctionTimeBeforeReset("WSTETH-A", 7200 seconds);
+        DssExecLib.setAuctionTimeBeforeReset("WSTETH-B", 7200 seconds);
+        DssExecLib.setAuctionTimeBeforeReset("ETH-B",    4800 seconds);
+        DssExecLib.setAuctionTimeBeforeReset("WBTC-B",   4800 seconds);
```

Comment:
Should the first `WBTC-B` be `WBTC-C`? Seems like there is a dup here and that you are missing the `WBTC-C -> 7,200 sec` from the spreadsheet

Reply:
Good catch.